docs/src/test-api/class-fullconfig.md

```diff
@@ -64,6 +64,12 @@ See [`property: TestConfig.maxFailures`].
 
 See [`property: TestConfig.metadata`].
 
+## property: FullConfig.populateGitInfo
```

Comment:
We try to not add new properties to `FullConfig` unless there is a strong need for it. Can you store it as `FullConfigInternal.populateGitInfo` instead ? Should be similar to `tsconfig`.